tests/test_utils.py

```diff
@@ -37,3 +37,9 @@ def test_from_base64url_uint(inputval, expected):
 def test_force_bytes_raises_error_on_invalid_object():
     with pytest.raises(TypeError):
         force_bytes({})  # type: ignore[arg-type]
+
+
+def test_is_ssh_key():
+    assert is_ssh_key(b"ecdsa-sha2-nistp256 any") is True
+    assert is_ssh_key(b"not a ssh key") is False
+    assert is_ssh_key(b"any-cert-v01@openssh.com any") is True
```

Comment:
I'm not sure this actually exists in the real world https://github.com/jpadilla/pyjwt/pull/940/files#r1460676243